### ISSUE TITLE
Create from Template lists no templates on iOS and iPadOS

### ISSUE THREAD

**Issue by wkjarosz** [ksalzke/templates-for-omnifocus#37]
First, thanks for the great plugin. It works great on macOS.

I haven’t been able to get the automation to work on iOS/iPadOS though. I have the plugin installed in my iCloud Drive so that it properly shows up on all three platforms, but when I click “Create from Template” it lists nothing (or rather “No Value”) in the dropdown to select the template. The “Create” button is also greyed out since I can’t select a template to start from. Any help welcome. Thanks again.

**Comment by ksalzke** [comment]
Hi @wkjarosz! 

This might be a silly question, but is this possibly the screen you're seeing?

![PNG image](https://user-images.githubusercontent.com/16893787/183287516-d139abbd-9e31-4a75-ae4f-0b8388c4adfb.png)

If so, you might just need to tap on the 'No Value' to expand the 'dropdown' in the Omni interface.

If not, could you perhaps post a screenshot of what you're seeing?

Thanks!
Kaitlin

**Comment by wkjarosz** [comment]
Yes, I do see the same screen (first acreenshot), but tapping the No value doesn't give me any templates to choose from in the dropdown menu (second screenshot).

![image](https://user-images.githubusercontent.com/10929749/183310277-5e83a280-7941-448d-bcfb-0c482e29b48c.png)

![image](https://user-images.githubusercontent.com/10929749/183310283-09096af1-dcf0-4c0a-b368-b367fd113dfa.png)
